utils/node_local_test.py

```diff
@@ -2671,6 +2671,10 @@ def test_cont_copy(self):
                'daos://{}/{}'.format(self.pool.uuid, self.container)]
         rc = run_daos_cmd(self.conf, cmd)
         print(rc)
+        lineresult = rc.stdout.decode('utf-8').splitlines()
+        assert lineresult[1] == '    Directories: 1'
+        assert lineresult[2] == '    Files:       1'
+        assert lineresult[3] == '    Links:       1'
```

Comment:
You could check len(lineresult) here as well.

Reply:
Added "assert len(lineresult) == 4".